CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Change history for ui-plugin-query-builder
 
+## [2.0.3](https://github.com/folio-org/ui-plugin-query-builder/tree/v2.0.3) (2025-04-18)
+
+* [UIPQB-204](https://folio-org.atlassian.net/browse/UIPQB-204) Add plugin-find-organization for "Organization — Name" and "Organization — Code" (additional work)
+* [UIPQB-208](https://folio-org.atlassian.net/browse/UIPQB-207) Resolve an error where multiple dropdown values would result in invalid user-friendly queries
+* [UIPQB-208](https://folio-org.atlassian.net/browse/UIPQB-208) Don't display invalid fields in lists
+* [UIPQB-206](https://folio-org.atlassian.net/browse/UIPQB-209) Add user-friendly query display to `ResultViewer`
+
 ## [2.0.2](https://github.com/folio-org/ui-plugin-query-builder/tree/v2.0.2) (2025-04-10)
 
 * [UIPQB-204](https://folio-org.atlassian.net/browse/UIPQB-204) Add plugin-find-organization for "Organization — Name" and "Organization — Code"
 * [UIPQB-206](https://folio-org.atlassian.net/browse/UIPQB-206) Expose `setIsModalShown` prop for consumers to know when the modal is open
-* [UIPQB-208](https://folio-org.atlassian.net/browse/UIPQB-208) Don't display invalid fields in lists
```

Comment:
this was added as part of pr introducing uipqb-208, but this was after 2.0.2 was released 🤔